app/views/kaui/accounts/_billing_info.html.erb

```diff
@@ -110,28 +110,48 @@
               <% end %>
             </table>
 
-            <% if can? :trigger, Kaui::Invoice %>
+
+             <% if can? [:trigger, :dry_run], Kaui::Invoice %>
                 <div style="padding-bottom: 50px;">
                   <%= form_tag kaui_engine.trigger_invoice_path(params.to_h), :method => :post do %>
                       <div class="form-group">
                         <%= label_tag :target_date, 'Trigger invoice generation', :class => 'col-sm-6 control-label', :style => 'padding-left: 0;' %>
                         <div class="col-sm-3">
-                          <input class="form-control" id="target_date" name="target_date" type="text" data-provide="datepicker" data-date-format="yyyy-mm-dd" data-date-today-highlight="true">
+                          <input class="form-control" name="target_date" type="text" data-provide="datepicker" data-date-format="yyyy-mm-dd" data-date-today-highlight="true">
                         </div>
-                        <% if can? :dry_run, Kaui::Invoice %>
                           <div style="width: 14.5%; float: left; padding: 0;">
                             <%= label_tag :dry_run do %>
                                 <%= check_box_tag :dry_run, '1', true %>&nbsp;<small>Dry-run</small>
                             <% end %>
                           </div>
-                        <% end %>
                         <div class="col-xs-1" style="padding: 0;">
                           <%= button_tag '<i class="fa fa-magic"></i>'.html_safe, :class => 'btn btn-xs' %>
                         </div>
                       </div>
                   <% end %>
                 </div>
+            <% elsif can? :dry_run, Kaui::Invoice %>
```

Comment:
Not sure you need the `elsif`. Just doing `if can?(:trigger, Kaui::Invoice) || can?(: dry_run, Kaui::Invoice)` should do it above.